### ISSUE TITLE
[FR] control crafting menu sort

### ISSUE THREAD

**Issue by ividyon** [Dasaav-dsv/libER#34]
when you press L3 in the crafting menu, you can select an alternate sort type which includes the category separator bars from the regular inventory

this is reset every time you close and reopen the menu

controlling this via libER would be cool

**Comment by Dasaav-dsv** [comment]
Menu manipulation will be looked at a much later time (considering it's all Scaleform). It's heavily composition based and it's not yet clear to me how easy to access these things are going to be, since I am really putting this off until much later.

**Comment by ThomasJClark** [comment]
I do have this implemented transmog. It's not complete (my only use case is setting a specific sort type and direction for one shop) but it does have the memory location of this field in `CSMenuSystemSaveLoad`

https://github.com/ThomasJClark/elden-ring-transmog/blob/main/src/ertransmogrify_shop.cpp#L60-L70

**Comment by ThomasJClark** [comment]
it should be possible to figure out all the sort types with cheat engine, and probably write actual enums and structs for them instead of what I originally did

**Comment by ividyon** [comment]
yeah I think scaleform isn't as much of a factor as you'd think, they don't use any ActionScript or basically any flash-side logic whatsoever, it's all controlled by engine code
